README.md

```diff
@@ -33,12 +43,8 @@ Current unit tests have the following component:
 - A module with the testing code that compiles (and compares) the preconditions, build the compilation settings, compiles the contract and compares the outputs.
 
 
-## TODOs
+## Write your own BitMLx contracts
+
+If you want to write your own BitMLx contracts: 
 
-[x] Change priority choices to use global time formulas and implement timed priority choice.
-[x] Implement the stipulation protocol.
-[x] Implement the `reveal` primitive.
-[x] Implement the participant authorization (signature) primitive.
-[x] Adapt Chrysa's more complex examples to the new syntax.
-[x] (maybe) Write a well-formed checker to ensure that a contract is valid before compiling.
-[] (maybe) Build an integration script to further compile outputs to bitcoin transactions using the existing BitML compiler.
\ No newline at end of file
+- `stack haddock --no-haddock-deps bitmlx --open` build the documentation and opens it in your web browser. Probably a good entry point!
```

Comment:
Can you add some lines here specifying where to put your contract code and how to add it to main? Basically what you are doing now.